packages/web/docs/src/app/gateway/page.tsx

```diff
@@ -54,7 +56,14 @@ export default function HiveGatewayPage() {
       </Hero>
       <GatewayFeatureTabs className="relative mt-6 sm:mt-[-72px] sm:bg-blue-100" />
       <OrchestrateYourWay className="mx-4 mt-6 sm:mx-8" />
-      {/* Federation-Compatible Gateway Benchmarks */}
+      <ErrorBoundary
+        fallback={
+          // this section doesn't make sense if data didn't load, so we just unmount
+          null
```

Comment:
This is needed in case the `data.json` was down. The promise rejects, and then React rips everything until this error boundary and renders this null.